packages/neuron-ui/src/components/SpecialAssetList/hooks.ts

```diff
@@ -193,15 +194,18 @@ export const useSpecialAssetColumnInfo = ({
       const isLockedCheque = status === 'withdraw-asset' && targetTime && Date.now() < targetTime
       const isNFTTransferable = assetInfo.type === NFTType.NFT && assetInfo.data === 'transferable'
       const isNFTClassOrIssuer = assetInfo.type === NFTType.NFTClass || assetInfo.type === NFTType.NFTIssuer
+      const isSpore = assetInfo.type === SporeType.Spore
 
       if (assetInfo.type === NFTType.NFT) {
         amount = nftFormatter(type?.args)
         status = 'transfer-nft'
       } else if (isNFTClassOrIssuer || assetInfo.type === 'Unknown') {
         amount = t('special-assets.unknown-asset')
+      } else if (isSpore) {
+        amount = sporeFormatter(type?.args, item.data, item.customizedAssetInfo.data)
       }
```

Comment:
It would be better to transform multi branches `if-else` into `switch-case` for maintenance.

Reply:
The process not only depends on the type(kind) of a cell, e.g. the second branch `isNFTClassOrIssuer || assetInfo.type === 'Unknown'`, if we refactor it to `switch-case` style, the nested level will be increased, maybe the `if-else` is superior to the `switch-case` in this situation

Reply:
I didn't get `the nested level will be increased`, it would be as follow

```ts
switch (assetInfo.type) {
  case NFTType.NFT: {
    // ...
    break
  }
  case NFTType.Spore: {
    // ...
    break
  }
  case 'Unknown': {
    // ...
    break
  }
  default: {
    if (isNFTClassOrIssuer) {
      // ...
    }
  }
}
```
All complexity will be limited in the fallback branch

Reply:
according to the `const isNFTClassOrIssuer = assetInfo.type === NFTType.NFTClass || assetInfo.type === NFTType.NFTIssuer`, the second branch will be

```ts
switch(assetInfo.type) {
// ...
  case (NFTType.NFTClass):
  case (NFTType.NFTIssuer): {
    if(assetInfo.type === 'Unknown') {
      amount = ...
    }
  }
// ...
}
```